scenes/game/ai_player.gd

```diff
@@ -467,18 +468,26 @@ func get_strike_actions(game_logic : LocalGame, me : LocalGame.Player, _opponent
 		for card in me.hand:
 			if card.definition['gauge_cost'] > me.gauge.size():
 				# Skip cards we can't pay for.
+				# But remember them in case we have 0 options and must strike with something?
```

Comment:
It might also be helpful to add this logic for strikes from gauge? the current AI can't really make use of those abilities well at the moment but may be nice for the future